.github/workflows/crowdin-i18n-docs.download.yml

```
@@ -128,6 +128,40 @@ jobs:
           CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID_DOCS }}
           CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_SERVICE_TOKEN }}
 
+      ##### Download Italian #####
+      - name: Crowdin Download Italian  Translations
```

```suggestion
      - name: Crowdin Download Italian Translations
```
One very small nit-pick.